semantic_conventions/trace/instrumentation/aws-lambda.yaml

```
@@ -0,0 +1,61 @@
+groups:
+  - id: aws.lambda
+    brief: "Lambda function invocation spans."
+    attributes:
+      - ref: faas.execution
+        brief: "The value of the AWS Request ID from the Lambda `Context`."
+        examples:
+          - 943ad105-7543-11e6-a9ac-65e093327849
+      - ref: faas.id
+        brief: "The value of the invocation arn for the function from the Lambda `Context`."
+        examples:
+          - arn:aws:lambda:us-east-2:123456789012:function:my-function:1
+      - ref: cloud.account.id
+        brief: "The account ID for the function. If not provided on Lambda `Context`, it SHOULD be parsed from the value of `faas.id` as the fifth item when splitting on `:`."
```

That "brief" is not very brief

```suggestion
        brief: "The account ID for the function."
        note: >
          If not provided on Lambda `Context`, it SHOULD be parsed from the value of `faas.id`
          as the fifth item when splitting on `:`.
```